ChangeLog.md

```diff
@@ -1,3 +1,7 @@
+## 0.2.12-preview - April 2020
+#### Az.Blueprint
+* Added support for management group level Blueprint assignment
+
```

Comment:
We don't need to change the changelog.md for the preview release.

Reply:
Done